travis/python_tests.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+status=0
+
+function run_test {
```

Comment:
Personally I've found it helpful in xpro to have the command printed before we run it: https://github.com/mitodl/mitxpro/blob/master/travis/python_tests.sh#L4-L9